docs/openapi.yml

```diff
@@ -297,7 +294,6 @@ paths:
       description: Creates a multiple output dataset objects.
       tags:
         - Jobs
-      required: true
```

Comment:
Where these `required` fields not used?

Reply:
The location here doesn't support this field per the spec.

Reply:
👍